user_docs/en/changes.t2t

```diff
@@ -28,16 +28,28 @@ Please refer to [the developer guide https://www.nvaccess.org/files/nvda/documen
 
 - Note: this is an Add-on API compatibility breaking release.
 Add-ons will need to be re-tested and have their manifest updated.
-- Updated Comtypes to version 1.2.0. (#15513)
-- Updated fast_diff_match_patch to version 2.0.1. (#15514)
-- Updated SCons to version 4.5.2. (#15529)
 - Added extension point: ``treeInterceptorHandler.post_browseModeStateChange``. (#14969)
 -
 
 === API Breaking Changes ===
 These are breaking API changes.
 Please open a GitHub issue if your Add-on has an issue with updating to the new API.
 
+- NVDA is now built with Python 3.11 (#12064)

```

Comment:
```suggestion
- Updated Python to 3.11. (#12064)
```

Reply:
You made this suggestion previously here https://github.com/nvaccess/nvda/pull/15544#discussion_r1339474402 - it appears to be deleted now. The reasoning against it still applies. We wouldn't say "Updated to Visual Studio 2022", we would say "NVDA is now built with VS 2022".

Reply:
In the case of VS, this affects compiled (i.e. NVDAHelper) code, not interpreted Python run from an included DLL. Also we have used the "updated Python to x" language with all other updates (see 2019.3 and 2015.1 changes for developers).